src/main/scala/uk/gov/hmrc/githubclient/model.scala

```diff
@@ -208,6 +207,6 @@ case class ExtendedRepository(name: String,
                               fork: Boolean,
                               createdAt: Date,
                               pushedAt: Date,
-                              @SerializedName("private") isPrivate: Boolean,
+                              `private`: Boolean,
```

Comment:
This is the core change.

Reply:
How does serialization work in this repo? Is that internal to the Git client we use?

Reply:
that's right, ser/des is internal to the git client.